### PR TITLE
Gp 63 fe dasboard of a game

### PR DESCRIPTION
**Pull request by dg-panh**
**New Feature:**

- game dashboard
- list user (game)

### CODE REVIEW

**Review by thuanvmse161224 — DISMISSED** [review]
lg

File: src/Layout/GameDashboardLayout/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { Box, Stack, AppBar, Toolbar, IconButton, Tabs, Tab } from '@mui/material'
+
+import { Games as GamesComponent } from '../../../pages/Games'
+
+
+function samePageLinkNavigation(
+	event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+) {
+	if (
+		event.defaultPrevented ||
```
Comment (danielNg91) [inline src/Layout/GameDashboardLayout/Navbar/Navbar.tsx:11]: should be break into smaller functions

File: src/Layout/GameDashboardLayout/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		/>
+	)
+}
+
+export default function Navbar() {
+	const [value, setValue] = React.useState(1)
+
+	const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+		// event.type can be equal to focus with selectionFollowsFocus.
+		if (
```
Comment (danielNg91) [inline src/Layout/GameDashboardLayout/Navbar/Navbar.tsx]: break into smaller functions for reability

File: src/common/constant.ts [hunk 3]
```diff
@@ -75,6 +79,15 @@ export const PrivateRouters: Route[] = [
 			title: 'Users',
 		},
 	},
+	{
+		path: convertNameToPath(PageName.PLAYER),
```
Comment (danielNg91) [inline src/common/constant.ts]: remove this functions, use string for all paths

**Review by danielNg91 — CHANGES_REQUESTED** [review]

File: src/Layout/GameDashboardLayout/Navbar/Navbar.tsx (hunk 1 above)
Comment (dg-panh) [reply src/Layout/GameDashboardLayout/Navbar/Navbar.tsx:11]: It's so small already

File: src/Layout/GameDashboardLayout/Navbar/Navbar.tsx (hunk 2 above)
Comment (dg-panh) [reply src/Layout/GameDashboardLayout/Navbar/Navbar.tsx]: already small

File: src/Layout/GameDashboardLayout/Navbar/Navbar.tsx (hunk 1 above)
Comment (danielNg91) [reply src/Layout/GameDashboardLayout/Navbar/Navbar.tsx:11]: The code is so dirty, to check many conditions like this you should create a function to solve that

**Review by danielNg91 — APPROVED** [review]

**Pull request merged**